ui/apps/platform/src/types/cve.proto.ts

```diff
@@ -1,7 +1,66 @@
 export type VulnerabilitySeverity =
+    // | 'UNKNOWN_VULNERABILITY_SEVERITY'
     | 'LOW_VULNERABILITY_SEVERITY'
     | 'MODERATE_VULNERABILITY_SEVERITY'
     | 'IMPORTANT_VULNERABILITY_SEVERITY'
     | 'CRITICAL_VULNERABILITY_SEVERITY';
 
 export type VulnerabilityState = 'OBSERVED' | 'DEFERRED' | 'FALSE_POSITIVE';
+
+export type CVSSV2 = {
+    vector: string;
+    attackVector: AttackVectorV2;
+    accessComplexity: AccessComplexityV2;
+    authentication: AuthenticationV2;
+    confidentiality: ImpactV2;
+    integrity: ImpactV2;
+    availability: ImpactV2;
+    exploitabilityScore: number; // float
+    impactScore: number; // float
+    score: number; // float
+    severity: SeverityV2;
+};
+
+export type AccessComplexityV2 = 'ACCESS_HIGH' | 'ACCESS_MEDIUM' | 'ACCESS_LOW';
+
+export type AttackVectorV2 = 'ATTACK_LOCAL' | 'ATTACK_ADJACENT' | 'ATTACK_NETWORK';
```

Comment:
where did this values come from? I ask because https://www.first.org/cvss/v2/guide calls it access local, access adjacent, etc

Reply:
Adapted to TypeScript from https://github.com/stackrox/stackrox/blob/master/proto/storage/cve.proto#L97-L101